pages/agent/v3/hooks.md.erb

```diff
@@ -128,3 +168,50 @@ An example of a windows `environment.bat` hook:
 ECHO "--- \:house_with_garden\: Setting up the environment"
 SET GITHUB_RELEASE_ACCESS_KEY='xxx'
 ```
+
+## Agent Hook Locations
```

Comment:
I found the mapping slightly hard to parse here, although don't yet have a better suggestion.

Reply:
I’ve tweaked this a bit in my local changes to be pushed up, let me know what you think when I push it up